tests/CSharpTemp/CSharpTemp.h

```diff
@@ -175,8 +175,9 @@ class DLL_API P : Proprietor
 };
 
 // Tests destructors
-struct DLL_API TestDestructors
+class DLL_API TestDestructors                                  //Value Types can't have destructors
```

Comment:
@ddobrev @tritao I know I should not have changed this. But, I tried a few things but could not find how to check if a 'destructor' or a 'zero argument Constructor' has been "explicitly defined" by the user! Please help.